src/cmdlib.sh

```diff
@@ -247,18 +247,32 @@ prepare_build() {
     mkdir tmp && TMPDIR=$(pwd)/tmp
 }
 
+commit_overlay() {
+    local name path respath
+    name=$1
+    path=$2
+    respath=$(realpath "${path}")
+    echo -n "Committing ${name}: ${path} ... "
+    ostree commit --repo="${tmprepo}" --tree=dir="${respath}" -b "${name}" \
+        --owner-uid 0 --owner-gid 0 --no-xattrs --no-bindings --parent=none \
+        --timestamp "${git_timestamp}"
+}
+
 runcompose() {
     # Implement support for automatic local overrides:
     # https://github.com/coreos/coreos-assembler/issues/118
     local overridesdir=${workdir}/overrides
     local tmp_overridesdir=${TMPDIR}/override
     local override_manifest="${tmp_overridesdir}"/coreos-assembler-override-manifest.yaml
-
+    local ovl="${configdir}/overlay"
```

Comment:
should we deprecate this in favor of overlay.d and warn or do you think it's fine to just have both?